CMakeLists.txt

```diff
@@ -299,3 +300,12 @@ ENDIF()
 
 FEATURE_SUMMARY(WHAT ENABLED_FEATURES DESCRIPTION "Enabled features:")
 FEATURE_SUMMARY(WHAT DISABLED_FEATURES DESCRIPTION "Disabled features:")
+
+configure_package_config_file(
+    "git2Config.cmake.in" "${CMAKE_CURRENT_BINARY_DIR}/git2Config.cmake"
+    INSTALL_DESTINATION "lib/cmake/git2"
+)
+
+install(EXPORT git2-targets DESTINATION lib/cmake/git2)
+
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/git2Config.cmake" DESTINATION "lib/cmake/git2")
```

Comment:
No camel casing here, please.

Is there any prior art you can point me to here about nomenclature?  I feel like this should be `libgit2`, not `git2`.